docs/en/core/comments

```diff
@@ -0,0 +1,60 @@
+
+```html
+<!-- This is a single-line comment -->
+```
+
+For a comment that spans multiple lines, it's simplest to place the opening comment markup and the closing markup on their own lines, sandwiching the text of your comment in between:
+
+```html
+<!--
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+Nunc et lobortis enim. Suspendisse ac sagittis lectus, eget consequat odio. Aliquam congue molestie justo in ullamcorper.
+
+Nulla blandit dignissim erat, sit amet luctus odio luctus et. Etiam euismod libero et lectus finibus, ac euismod metus iaculis.
+-->
+```
+
+To make inserting comments dead simple, Zettlr includes a comment keyboard shortcut. Typing `Ctrl-Shift-c` will auto-complete the basic comment syntax, placing your cursor in the center, between the opening and closing angle brackets, so you can begin typing right away:
+
+```html
+![Creating a new comment with the keyboard shortcut.](../img/ConverttoComment.gif)
+```
+
+If you've already written a block of text that you'd like to turn into a comment, you can simply highlight it with your cursor and _then_ type the `Ctrl-Shift-c` shortcut to convert the selected passage into a comment:
+
+![Converting text to a comment with the keyboard shortcut.](../img/ConverttoComment.gif)
+
+!!! note
+
+    As a Markdown editor, Zettlr does not natively support margin comments like those used in Google Docs or Microsoft Word. If you are [collaborating with editors or co-authors on these platforms](https://docs.zettlr.com/en/core/import/#working-with-co-workers), you can use Zettlr's export feature to generate a `.docx` or `.odt` document for them to comment on. After reviewing and incorporating your collaborator's comments in a word processor, you can then re-import the resulting document into Zettlr to continue working on it in the app.
+
+## Doing More With Comments
+
+While Zettlr is a dedicated writing tool, it borrows useful features inspired by the powerful text editors developers use for writing code. Developers have not one, but _two_ important uses for comments. The first is, as you would expect, to leave helpful remarks in their code they're writing. When it comes time to run the code, these remarks — wrapped in their special comment markup — aren't treated as part of the program.
+
+This hints at the second way developers commonly use comments, which is to "comment out" sections of their code. In other words, developers will take a chunk of the code they've written and wrap it in markup that tells the browser — or whatever's running their program — to ignore it, treating it as a comment instead of part of the program. This is handy for lots of reasons. Among others, it's useful when a developer is thinking about getting rid of a chunk of code, but isn't ready to delete it just yet, possibly because they want to see how the program works without it or because they want to see the old code for reference as they rewrite part of the program.
```

Comment:
I'd argue we can get rid of everything after "…treating it as a comment instead of part of the program" to get back to the writer's use case for HTML comments faster.